lib/spack/docs/getting_started.rst

```diff
@@ -94,7 +94,7 @@ Spack has to install it on initial use, which is called bootstrapping.
 
 Spack provides two ways of bootstrapping ``clingo``: from pre-built binaries
 (default), or from sources. The fastest way to get started is to bootstrap from
-pre-built binaries.
+pre-built binaries downloaded from Spack mirrors. Bootstrapping from sources makes it possible to use Spack even in the presence of restrictions on the internet access (e.g., when the host is accessible only over SSH or if hosts have to be explicitly whitelisted).
```

Comment:
I would leave this sentence as it was. The rationale is that you can bootstrap from binaries even on air-gapped systems, since the binaries are freely available [here](https://github.com/alalazo/spack-bootstrap-mirrors/releases/download/v0.1-rc.2/bootstrap-buildcache.tar.gz). Using binaries or not is essentially a choice of trusting them or not.

Reply:
> The rationale is that you can bootstrap from binaries even on air-gapped systems, since the binaries are freely available here. Using binaries or not is essentially a choice of trusting them or not.

I do not want to trust binaries on systems that were intentionally air-gapped.

Reply:
Two comments:
1. I'm not saying you _should_ I'm saying you _can_ 
2. You don't necessarily have to trust github binaries, but sites can e.g. produce their own

From what you write it seems that using binaries is not possible on air-gapped systems.